sherlock/sherlock.py

```diff
@@ -583,7 +584,14 @@ def main():
     if args.tor or args.unique_tor:
         print("Using Tor to make requests")
         print("Warning: some websites might refuse connecting over Tor, so note that using this option might increase connection errors.")
-
+    
+    if args.no_color:
+        # Disable color output.
+        init(strip=True, convert=False)
+    else:
+        # Enable color output.
+        init(autoreset=True)
+        
```

Comment:
Would this be a better solution?
```python
if not args.no_color:
    init(autoreset=True)
```

So by default color is on. If `--no-color` is used, then `init(autoreset=True)` is skipped because of the condition in the if statement.

Reply:
On my machine (OS X) that doesn't work. Even with the no-color flag, colored output is displayed. The reason for this behavior is that [init()](https://github.com/tartley/colorama#description) converts ANSI characters to win32 calls. That has no effect on  Linux/OS X because there,  colorama does nothing.